src/NUnitFramework/tests/Attributes/CancelAfterTests.cs

```diff
@@ -294,7 +294,7 @@ public void TestWithCancelAfterAttributeAndTestCaseSourceHasCancellationToken(in
 #pragma warning restore NUnit1003 // The TestCaseAttribute provided too few arguments
 #pragma warning restore NUnit1027 // The test method has parameters, but no arguments are supplied by attributes
 #pragma warning restore NUnit1029 // The number of parameters provided by the TestCaseSource does not match the number of parameters in the Test method
-
+#pragma warning disable NUnit1029
```

Comment:
It seems odd to disable this when we have just restore it on the line above. Also, I would have thought that this work now?

Reply:
No, it doesn't.  It still fails, and I thought to create an issue for it, but from your comment - was it supposedly fixed ?

Reply:
but I'll delete 296 and 297